Real-Time Corruptor/BizHawk_RTC/BizHawk.Client.EmuHawk/RTC/RTC_NetCore.cs

```diff
@@ -386,8 +386,7 @@ public void StopNetworking(bool fromBye = false, bool stayConnected = false)
 				return;
 			}
 
-			if (!stayConnected)
-				expectingSomeone = false;
+			expectingSomeone &= stayConnected;
```

Comment:
While it does save a line, I find unnecessary bitwise operations to be more annoying in the long run, especially when it's going to compile to the same code either way.